data/json/items/comestibles/mutagen.json

```diff
@@ -211,8 +211,6 @@
         "description" : "A super-concentrated stem cell treatment.  You need a syringe to inject it.",
         "price" : 900000, "//" : "*May* have been commercially traded, but not in this preparation.",
         "color" : "magenta",
-        "healthy" : -10,
```

Comment:
Strict mode rejects useless assignments - in this case that item redefines `stim` and `healthy`as the same value it was already set at - most probably because someone forgot to remove them when implementing that file.